src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1247,4 +1247,10 @@ This version used in: {2}</value>
   <data name="default_expression_can_be_simplified" xml:space="preserve">
     <value>'default' expression can be simplified</value>
   </data>
+  <data name="Format_string_0_contains_invalid_placeholder_1" xml:space="preserve">
+    <value>Format string '{0}' contains invalid placeholder '{1}'</value>
```

Comment:
I'm curious how this shows to the user.  The format string itself is effectively unbounded in size.  So i would not want it embedded in the error string.  I would expect to just point at the placeholder that is bad and just call that part out.